docs/sources/query/logcli.md

```diff
@@ -51,7 +51,7 @@ You can set up tab-completion for `logcli` with one of the two options, dependin
   eval "$(logcli --completion-script-zsh)"
   ```
 
-## LogCLI usage
+## Usage
```

Comment:
Sorry but I'm going to have to ask that we change a few of these headings back.  Because the Loki docs are part of the greater Grafana Docs website, for search engine optimization, we sometimes do need to avoid common headings like "usage" and "command reference" and be more specific.
```suggestion
## LogCLI Usage
```